movielist/src/components/features/auth/RegisterForm/RegisterForm.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+
+import { mutator } from "@/fetchers/mutators";
+import { Button, chakra, Flex, FormControl, Heading, Input, Link, Text } from "@chakra-ui/react";
+import { useRouter } from "next/navigation";
+import { useForm } from "react-hook-form";
+import useSWRMutation from "swr/mutation";
+
+export interface IRegisterFormInputs {
+    email: string,
+    password: string,
+    password_confirmation: string
+}
+
+export function RegisterForm() {
+    const router = useRouter();
+    const {register, handleSubmit} = useForm<IRegisterFormInputs>();
+    const {trigger} = useSWRMutation("https://tv-shows.infinum.academy/users", mutator)
+    const onRegister = async (data: IRegisterFormInputs) => {
+        if (data.password.length < 8){
+            window.alert("Password is not long enough");
+            return;
+        }
+        if (data.password !== data.password_confirmation) {
+            window.alert("Passwords dont match");
+            return;
+        }
+        await trigger(data).then(() => {
+            router.push("/login");
+        });
```

Comment:
```suggestion
        await trigger();
```

A onda bih onSuccess odradio šta treba (redirectao)

Reply:
prebacio router push na onSuccess